[2021-04-25 14:02:07,205] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.delete_cluster_dataproc 2021-04-25T13:00:00+00:00 [queued]>
[2021-04-25 14:02:07,236] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.delete_cluster_dataproc 2021-04-25T13:00:00+00:00 [queued]>
[2021-04-25 14:02:07,237] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-25 14:02:07,239] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-04-25 14:02:07,240] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-25 14:02:07,249] {taskinstance.py:1063} INFO - Executing <Task(PatchedDataprocDeleteClusterOperator): delete_cluster_dataproc> on 2021-04-25T13:00:00+00:00
[2021-04-25 14:02:07,254] {standard_task_runner.py:52} INFO - Started process 1392 to run task
[2021-04-25 14:02:07,258] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'recent_played_ingestion', 'delete_cluster_dataproc', '2021-04-25T13:00:00+00:00', '--job-id', '112', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/recent_played_ingestion.py', '--cfg-path', '/tmp/tmpxmgg4nzc', '--error-file', '/tmp/tmpezysv3bq']
[2021-04-25 14:02:07,262] {standard_task_runner.py:77} INFO - Job 112: Subtask delete_cluster_dataproc
[2021-04-25 14:02:07,321] {logging_mixin.py:104} INFO - Running <TaskInstance: recent_played_ingestion.delete_cluster_dataproc 2021-04-25T13:00:00+00:00 [running]> on host 67976f37379c
[2021-04-25 14:02:07,370] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=recent_played_ingestion
AIRFLOW_CTX_TASK_ID=delete_cluster_dataproc
AIRFLOW_CTX_EXECUTION_DATE=2021-04-25T13:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-25T13:00:00+00:00
[2021-04-25 14:02:07,383] {dataproc.py:840} INFO - Deleting cluster: ephemeral-recent-played-spark-cluster-20210425
[2021-04-25 14:02:35,514] {dataproc.py:852} INFO - Cluster deleted.
[2021-04-25 14:02:35,535] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=recent_played_ingestion, task_id=delete_cluster_dataproc, execution_date=20210425T130000, start_date=20210425T140207, end_date=20210425T140235
[2021-04-25 14:02:35,589] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-04-25 14:02:35,646] {local_task_job.py:146} INFO - Task exited with return code 0
